### PR TITLE
feature/WDP190703-17

### PR DESCRIPTION
**Pull request by danielmark0116**
Add filtering through the categories in NEW FURNITURE section.

Cherry-picked mixins and variables from my previous task

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -21,5 +21,6 @@
 
     <script src="./scripts/bootstrap.min.js"></script>
     <script src="./scripts/app.js"></script>
+    <script src="./scripts/tabs.js"></script>
```
Comment (matwjablonski) [inline src/index.html]: use import

File: src/scripts/tabs.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+let filterValue = 'table';
+
+// FUNCTIONS:
```
Comment (matwjablonski) [inline src/scripts/tabs.js]: unnecessary comment

File: src/scripts/tabs.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+// clearing active classes from tabs
```
Comment (matwjablonski) [inline src/scripts/tabs.js]: your code should be self commented and self described - this comments are unnecessary

File: src/index.html (hunk 1 above)
Comment (danielmark0116) [reply src/index.html]: Fixed :)

File: src/scripts/tabs.js (hunk 2 above)
Comment (danielmark0116) [reply src/scripts/tabs.js]: Fixed :)

File: src/scripts/tabs.js (hunk 3 above)
Comment (danielmark0116) [reply src/scripts/tabs.js]: Fixed :)

File: src/sass/components/_new-furniture-list.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.new-furniture-list {
+  @include fade-long;
+  opacity: 0;
+  display: none;
+  &.show {
```
Comment (matwjablonski) [inline src/sass/components/_new-furniture-list.scss]: add empty line before

File: src/sass/components/_new-furniture-list.scss [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    opacity: 1;
+  }
+  &.fadeOut {
```
Comment (matwjablonski) [inline src/sass/components/_new-furniture-list.scss]: add empty line before

File: src/scripts/app.js [hunk 6]
```diff
@@ -1 +1,4 @@
 console.log('hello');
+
+// IMPORT -> script with TABS filtering through New Furniture Lists Categories
```
Comment (matwjablonski) [inline src/scripts/app.js]: `import './tabs';` - should works

File: src/scripts/app.js [hunk 7]
```diff
@@ -1 +1,4 @@
 console.log('hello');
```
Comment (matwjablonski) [inline src/scripts/app.js]: remove console.log

File: src/scripts/app.js (hunk 6 above)
Comment (matwjablonski) [inline src/scripts/app.js]: remove comment

File: src/scripts/tabs.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
+module.exports = function () {
+  let filterValue = 'table';
+
+  // function that removes "active" classes from TAB BUTTONS (given that they have such class at all)
```
Comment (matwjablonski) [inline src/scripts/tabs.js]: these comments are unnecessary - your code should be self commented and self described

File: src/scripts/tabs.js [hunk 9]
```diff
@@ -0,0 +1,71 @@
+module.exports = function () {
+  let filterValue = 'table';
```
Comment (matwjablonski) [inline src/scripts/tabs.js]: why `let`?

File: src/scripts/tabs.js [hunk 10]
```diff
@@ -0,0 +1,71 @@
+module.exports = function () {
```
Comment (matwjablonski) [inline src/scripts/tabs.js]: this is code from `common js` - unnecessary

File: src/scripts/tabs.js [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  // Adding active class to the selected tab
+  function activeTab (item) {
+    document.querySelector(`[data-category=${item}]`).classList.add('active');
+    filterValue = item.toString();
+    listToggle();
+  }
+
+  // ---------------------------------------------------------------------------------------------------------
```
Comment (matwjablonski) [inline src/scripts/tabs.js]: ??

File: src/sass/components/_new-furniture-list.scss (hunk 4 above)
Comment (danielmark0116) [reply src/sass/components/_new-furniture-list.scss]: Fixed :)

File: src/sass/components/_new-furniture-list.scss (hunk 5 above)
Comment (danielmark0116) [reply src/sass/components/_new-furniture-list.scss]: Fixed :)

File: src/scripts/app.js (hunk 6 above)
Comment (danielmark0116) [reply src/scripts/app.js]: Fixed :)

File: src/scripts/app.js (hunk 7 above)
Comment (danielmark0116) [reply src/scripts/app.js]: Fixed :)

File: src/scripts/app.js (hunk 6 above)
Comment (danielmark0116) [reply src/scripts/app.js]: Fixed :)

File: src/scripts/tabs.js (hunk 8 above)
Comment (danielmark0116) [reply src/scripts/tabs.js]: Fixed :)

File: src/scripts/tabs.js (hunk 9 above)
Comment (danielmark0116) [reply src/scripts/tabs.js]: It is being reassigned inside activeTab function

File: src/scripts/tabs.js (hunk 11 above)
Comment (danielmark0116) [reply src/scripts/tabs.js]: Fixed :)

**Pull request merged**